src/Forms/Validator.php

```diff
@@ -69,9 +70,14 @@ public static function formatMessage(Rule $rule, bool $withValue = true)
 			static $i = -1;
 			switch ($m[1]) {
 				case 'name': return $rule->control->getName();
-				case 'label': return $rule->control instanceof Controls\BaseControl
-					? rtrim($rule->control->translate($rule->control->getCaption()), ':')
-					: null;
+				case 'label':
+					if ($rule->control instanceof Controls\BaseControl) {
```

Comment:
I think this way (replacing ternary operator with if with early return)  is more readable intead of having 2 nested ternary operators, but i am opened to adapt it to desired codestyle needs - there is missing some coding standard tool to force code style :-).